helix-view/src/gutter.rs

```diff
@@ -1,21 +1,53 @@
 use std::fmt::Write;
 
 use crate::{
+    editor::GutterType,
     graphics::{Color, Style, UnderlineStyle},
     Document, Editor, Theme, View,
 };
 
+fn count_digits(n: usize) -> usize {
+    // NOTE: if int_log gets standardized in stdlib, can use checked_log10
```

Comment:
Looks like this entered the FCP the other day https://github.com/rust-lang/rust/issues/70887#issuecomment-1289028216

Could you link the issue in this comment too to raise the chances of it being refactored later?